### DISCUSSION TITLE
Happy New Year!

### DISCUSSION THREAD

**Discussion by Meme-Eater** [Monika-After-Story/MonikaModDev#8894] [appreciation]
Not really an issue. I just wanted to wish the dev team and everyone else here a Happy Near Year. 2020 sucked, but now 2021 looks bright and hopeful.

Also, thanks for making Monika what it is today. I took her out to my family and we had fun :)

Good wishes for this new year!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
